var/spack/repos/builtin/packages/igraph/package.py

```diff
@@ -6,12 +6,42 @@
 from spack.package import *
 
 
-class Igraph(AutotoolsPackage):
+class Igraph(CMakePackage, AutotoolsPackage):
     """igraph is a library for creating and manipulating graphs."""
 
     homepage = "https://igraph.org/"
     url = "https://github.com/igraph/igraph/releases/download/0.7.1/igraph-0.7.1.tar.gz"
 
+    version("0.10.6", sha256="99bf91ee90febeeb9a201f3e0c1d323c09214f0b5f37a4290dc3b63f52839d6d")
     version("0.7.1", sha256="d978030e27369bf698f3816ab70aa9141e9baf81c56cc4f55efbe5489b46b0df")
 
+    build_system(
+        conditional("cmake", when="@0.10.6"),
+        conditional("autotools", when="@0.7.1"),
```

Comment:
Would be nice to know what range of versions support which build systems. Makes it easier to add new versions in the future.

Reply:
0.9 and later use CMake.